src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -0,0 +1,194 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {openInviteMembersModal} from 'app/actionCreators/modal';
+import {Panel} from 'app/components/panels';
+import {t} from 'app/locale';
+import AsyncView from 'app/views/asyncView';
+import Badge from 'app/components/badge';
+import Button from 'app/components/button';
+import InlineSvg from 'app/components/inlineSvg';
+import ListLink from 'app/components/links/listLink';
+import NavTabs from 'app/components/navTabs';
+import routeTitleGen from 'app/utils/routeTitle';
+import SentryTypes from 'app/sentryTypes';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import space from 'app/styles/space';
+
+type Props = AsyncView['props'] & {
+  children?: any;
+};
+
+class OrganizationMembersWrapper extends AsyncView<Props, AsyncView['state']> {
+  static contextTypes = {
+    router: PropTypes.object,
+    organization: SentryTypes.Organization,
+  };
+
+  getEndpoints(): ([string, string, any] | [string, string])[] {
+    const {orgId} = this.props.params;
+    return [
+      ['inviteRequests', `/organizations/${orgId}/invite-requests/`],
+      ['requestList', `/organizations/${orgId}/access-requests/`],
+    ];
+  }
+
+  getTitle() {
+    const {orgId} = this.props.params;
+    return routeTitleGen(t('Members'), orgId, false);
+  }
+
+  get hasExperiment() {
+    const {organization} = this.context;
+
+    if (!organization || !organization.experiments) {
+      return false;
+    }
+    return (
+      organization.experiments.JoinRequestExperiment === 1 ||
+      organization.experiments.InviteRequestExperiment === 1
+    );
+  }
+
+  get hasWriteAccess() {
+    const {organization} = this.context;
+    if (!organization || !organization.access) {
+      return false;
+    }
+    return organization.access.includes('member:write');
+  }
+
+  get showInviteRequests() {
+    return this.hasWriteAccess && this.hasExperiment;
+  }
+
+  get showNavTabs() {
+    const {requestList} = this.state;
+
+    // show the requests tab if there are pending team requests,
+    // or if the organization is exposed to the experiment and
+    // the user has access to approve or deny requests
+    return requestList.length > 0 || this.showInviteRequests;
+  }
+
+  get requestCount() {
+    const {requestList, inviteRequests} = this.state;
+    let count = requestList.length;
+
+    // if the user can't see the invite requests panel,
+    // exclude those requests from the total count
+    if (this.showInviteRequests) {
+      count += inviteRequests.length;
+    }
+    return count ? count.toString() : null;
+  }
+
+  updateRequestList = (id: string) => {
```

Comment:
NIT: You can leave off the `{` and `}` on this and it will indent a bit less, since it's only one expression inside the body.